app/templates/gulpfile.babel(gulp).js

```diff
@@ -392,7 +395,7 @@ gulp.task('build:client', ['transpile:client', 'styles', 'html'], () => {
 
     let assets = plugins.useref.assets({searchPath: ['client', '.tmp']});
 
-    return gulp.src(paths.mainView)<% if(filters.jade) { %>
```

Comment:
split this change out to a separate commit, since it's unrelated to the other changes